src/containers/Accounts/AccountsRouter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { useParams } from 'react-router'
+import { connect } from 'react-redux'
+import { AMMAccounts } from './AMM/AMMAccounts'
+import SocketContext from '../shared/SocketContext'
+import { getAccountInfo } from '../../rippled/lib/rippled'
+import NoMatch from '../NoMatch'
+import { Accounts } from './index'
+import { ERROR_MESSAGES } from './Errors'
+import Loader from '../shared/components/Loader'
+import { ACCOUNT_FLAGS } from '../../rippled/lib/utils'
+
+const getErrorMessage = (error: any) =>
+  ERROR_MESSAGES[error] || ERROR_MESSAGES.default
+
+function renderError(error: any) {
+  const message = getErrorMessage(error.code)
+  return (
+    <div className="accounts-page">
+      <NoMatch title={message.title} hints={message.hints} />
+    </div>
+  )
+}
+
+const AccountsRouter = () => {
+  const { id: accountId } = useParams<{ id: string }>()
+  const rippledSocket = useContext(SocketContext)
+  const [error, setError] = useState<any>()
+  const [comp, setComp] = useState<any>()
```

Comment:
These should be typed better

Reply:
It won't let me type these - results in an error: Argument of type 'Element' is not assignable to parameter of type 'SetStateAction '